Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/AttributesSectionTranslator.swift

```diff
@@ -0,0 +1,84 @@
+    func translateSection(
+        for symbol: Symbol,
+        renderNode: inout RenderNode,
+        renderNodeTranslator: inout RenderNodeTranslator
+    ) -> VariantCollection<CodableContentSection?>? {
+        translateSectionToVariantCollection(
+            documentationDataVariants: symbol.attributesVariants
+        ) { _, attributes in
+            guard !attributes.isEmpty else { return nil }
+            
+            func translateFragments(_ fragments: [SymbolGraph.Symbol.DeclarationFragments.Fragment]) -> [DeclarationRenderSection.Token] {
+                return fragments.map { fragment in
+                    let reference: ResolvedTopicReference?
+                    if let preciseIdentifier = fragment.preciseIdentifier,
+                       let resolved = renderNodeTranslator.context.localOrExternalReference(symbolID: preciseIdentifier)
+                    {
+                        reference = resolved
+                        renderNodeTranslator.collectedTopicReferences.append(resolved)
+                    } else {
+                        reference = nil
+                    }
+
+                    // Add the declaration token
+                    return DeclarationRenderSection.Token(fragment: fragment, identifier: reference?.absoluteString)
+                }
+            }
+            
+            return AttributesRenderSection(
+                title: "Attributes",
+                attributes: attributes.compactMap { kind, attribute in
+                    
+                    switch (kind, attribute) {
+                    case (.minimum, let value as SymbolGraph.AnyNumber):
+                        return RenderAttribute.minimum(String(value))
+                    case (.maximum, let value as SymbolGraph.AnyNumber):
+                        return RenderAttribute.maximum(String(value))
+                    case (.minimumExclusive, let value as SymbolGraph.AnyNumber):
+                        return RenderAttribute.minimumExclusive(String(value))
+                    case (.maximumExclusive, let value as SymbolGraph.AnyNumber):
+                        return RenderAttribute.maximumExclusive(String(value))
+                    case (.minimumLength, let value as Int):
+                        return RenderAttribute.minimumLength(String(value))
+                    case (.maximumLength, let value as Int):
+                        return RenderAttribute.maximumLength(String(value))
+                    case (.default, let value as SymbolGraph.AnyScalar):
+                        return RenderAttribute.default(String(value))
+                    case (.allowedTypes, let types as [SymbolGraph.Symbol.TypeDetail]):
+                        let tokens = types.compactMap {
+                            if let fragments = $0.fragments {
+                                return translateFragments(fragments)
+                            } else {
+                                return nil
+                            }
```

Comment:
Minor: This is a good use of `Optional.map`

```suggestion
                            $0.fragments.map(translateFragments)
```